[2025-04-06T17:47:04.794+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_prediction_dag.predict_fraud scheduled__2025-04-05T00:00:00+00:00 [queued]>
[2025-04-06T17:47:04.800+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_prediction_dag.predict_fraud scheduled__2025-04-05T00:00:00+00:00 [queued]>
[2025-04-06T17:47:04.801+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-06T17:47:04.814+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): predict_fraud> on 2025-04-05 00:00:00+00:00
[2025-04-06T17:47:04.817+0000] {standard_task_runner.py:60} INFO - Started process 76 to run task
[2025-04-06T17:47:04.820+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_prediction_dag', 'predict_fraud', 'scheduled__2025-04-05T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/fraud_pipeline_dag.py', '--cfg-path', '/tmp/tmp0yjzvc49']
[2025-04-06T17:47:04.822+0000] {standard_task_runner.py:88} INFO - Job 2: Subtask predict_fraud
[2025-04-06T17:47:04.861+0000] {task_command.py:423} INFO - Running <TaskInstance: fraud_prediction_dag.predict_fraud scheduled__2025-04-05T00:00:00+00:00 [running]> on host 1c22d88631dc
[2025-04-06T17:47:04.914+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fraud_prediction_dag' AIRFLOW_CTX_TASK_ID='predict_fraud' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-05T00:00:00+00:00'
[2025-04-06T17:47:11.026+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-06T17:47:11.032+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fraud_prediction_dag, task_id=predict_fraud, execution_date=20250405T000000, start_date=20250406T174704, end_date=20250406T174711
[2025-04-06T17:47:11.047+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-06T17:47:11.054+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-06T18:03:38.848+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_prediction_dag.predict_fraud scheduled__2025-04-05T00:00:00+00:00 [queued]>
[2025-04-06T18:03:38.853+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_prediction_dag.predict_fraud scheduled__2025-04-05T00:00:00+00:00 [queued]>
[2025-04-06T18:03:38.854+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-06T18:03:38.868+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): predict_fraud> on 2025-04-05 00:00:00+00:00
[2025-04-06T18:03:38.871+0000] {standard_task_runner.py:60} INFO - Started process 72 to run task
[2025-04-06T18:03:38.874+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_prediction_dag', 'predict_fraud', 'scheduled__2025-04-05T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/fraud_pipeline_dag.py', '--cfg-path', '/tmp/tmpo1gwqib7']
[2025-04-06T18:03:38.875+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask predict_fraud
[2025-04-06T18:03:38.911+0000] {task_command.py:423} INFO - Running <TaskInstance: fraud_prediction_dag.predict_fraud scheduled__2025-04-05T00:00:00+00:00 [running]> on host edc87b335a79
[2025-04-06T18:03:38.963+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fraud_prediction_dag' AIRFLOW_CTX_TASK_ID='predict_fraud' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-05T00:00:00+00:00'
[2025-04-06T18:03:45.517+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-06T18:03:45.523+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fraud_prediction_dag, task_id=predict_fraud, execution_date=20250405T000000, start_date=20250406T180338, end_date=20250406T180345
[2025-04-06T18:03:45.549+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-06T18:03:45.557+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
